src/main/kotlin/no/nav/syfo/client/Norg2Client.kt

```diff
@@ -29,17 +34,24 @@ class Norg2Client(
                 configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
             }
         }
+        expectSuccess = false
     }
 
     suspend fun getLocalNAVOffice(geografiskOmraade: String, diskresjonskode: String?): Enhet =
             retry("find_local_nav_office") {
-                client.get<Enhet>("$endpointUrl/enhet/navkontor/$geografiskOmraade") {
+                val httpResponse = client.get<HttpResponse>("$endpointUrl/enhet/navkontor/$geografiskOmraade") {
                     accept(ContentType.Application.Json)
                     contentType(ContentType.Application.Json)
                     if (!diskresjonskode.isNullOrEmpty()) {
                         parameter("disk", diskresjonskode)
                     }
                 }
+                if (httpResponse.status == NotFound) {
```

Comment:
Nice! Burde det være `warn`? Er det noe vi burde ha en metrikk på?

Reply:
Dette er normal oppførsel for alle brukere som bor i utlandet, er det noe vi som utviklere trenger å legge spesielt merke til..? Egentlig er dette en funksjonell feil :)

Reply:
En tanke: Hvis poden deres forsvinner vil kubernetes returnere 404 for alle requester om jeg ikke tar helt feil. Det _kan_ skape en del hodebry? :(

Reply:
Eller det er kanskje bare på port 80. Blir kanskje 503